flixel/input/gamepad/id/MFiID.hx

```diff
@@ -25,8 +25,8 @@ class MFiID
 	public static inline var DPAD_LEFT:Int = 19;
 	public static inline var DPAD_RIGHT:Int = 20;
 	
-	public static var LEFT_ANALOG_STICK(default, null) = new FlxGamepadAnalogStick(0, 1);
-	public static var RIGHT_ANALOG_STICK(default, null) = new FlxGamepadAnalogStick(2, 3);
+	public static var LEFT_ANALOG_STICK (default, null) = new FlxGamepadAnalogStick(0, 1, {up:21, down:22, left:23, right:24});
```

Comment:
In that case, wouldn't it be better if they weren't so close to the actual IDs? Maybe have them all start at 100-something an an indicator that they aren't real IDs?